mod_ci/controllers.py

```diff
@@ -442,6 +442,24 @@ def kvm_processor(db, kvm_name, platform, repository, delay):
             platform=platform, id=test.id))
 
 
+def ci_badge(state):
+    if test_type != TestType.pull_request:
```

Comment:
This won't work.

Reply:
okay
will find another way to figure them out

Reply:
I'm talking just about this. test_type is undefined in this context.

Reply:
the right one is test.test_type (watching from the code below(other function))

Reply:
ohh
i must add test_type as a param